Add ivermectin to therapeutics (#1002)
greenelab/covid19-review: content/20.pharmaceuticals.md

The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.

> The WHO guidelines [@doi:10.1136/bmj.m3379] and a systematic review [@doi:10.1136/bmj.m2980] are complementary living documents that summarize COVID-19 therapeutics.

Author: This is from #992, I messed up the merge commit

More recent research has indicated that ivermectin might function as a broad-spectrum antiviral by distrupting the trafficking of viral proteins by both RNA and DNA viruses [@doi:10.1038/s41429-020-0336-z; @doi:10.1038/ja.2017.11; @doi:10.1186/s12941-020-00368-w], although most of these studies have demonstrated this effect _in vitro_ [@doi:10.1186/s12941-020-00368-w].
The potential for antiviral effects on SARS-CoV-2 were investigated _in vitro_, and ivermectin was found to inhibit viral replication in a cell line derived from Vero cells (Vero-hSLAM) [@doi:10.1016/j.antiviral.2020.104787] but at a concentration that is much higher than that explored by existing dosage guidelines [@doi:10.4269/ajtmh.20-0271; @doi:10.1002/cpt.1889] and likely to be associated with significant side effects due to the increased potential that the compound could cross the mammalian blood-brain barrier [@doi:10.1080/13102818.2020.1775118; @doi:10.1371/journal.pntd.0001883].

> Retrospective studies and small RCTs began investigating the effects of standard doses of this low-cost, widely available drug.

Reviewer: We may not introduce the RCT acronym until later (maybe the summary table)?

Author: Good catch! Now addressed in #1001

One retrospective study reported that ivermectin reduced all-cause mortality [@doi:0.1016/j.chest.2020.10.009] while another reported no difference in clinical outcomes or viral clearance [@doi:10.1371/journal.pone.0242184].
Small RCTs enrolling less than 50 patients per arm have also reported a wide array of positive [@doi:10.4103/1995-7645.318304; @doi:10.1093/qjmed/hcab035; @doi:10.1016/j.toxrep.2021.03.003; @doi:10.1016/j.ijid.2020.11.191; @doi:10.1016/j.clinthera.2021.04.007] and negative results [@doi:10.1016/j.eclinm.2020.100720; @doi:10.21203/rs.3.rs-191648].
A slightly larger RCT enrolling 115 patients in two arms reported inconclusive results [@doi:10.18433/jpps32105].

> Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:https://doi.org/10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].

Reviewer 2: a meta-analyses has also had to be corrected as a result of the retracted studies. - https://doi.org/10.1093/ofid/ofab358 hard to squeeze it in though, might be best to leave it out.

Author: Great suggestion! Added below in a479a18f

> Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:https://doi.org/10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].

Reviewer: ```suggestion
Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].
```

I wanted to check how the retraction would render.  The authors aren't there, which makes sense, and it looks reasonable to me:
1. **Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic**
Research Square Platform LLC *Research Square Platform LLC* (2020-10-30) <https://doi.org/gmhmfm>
DOI: [10.21203/rs.3.rs-100956/v4](https://doi.org/10.21203/rs.3.rs-100956/v4)

> A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms [@doi:10.1001/jama.2021.3071], and although meta-analyses have reported both null [@doi:10.1101/2021.05.21.2125759; @doi:10.1101/2021.01.26.21250420] and beneficial [@doi:10.1016/j.dsx.2021.102186; @doi:10.1097/MJT.0000000000001402; @doi:10.1002/rmv.2265; @doi:10.31219/osf.io/k37ft; @doi:10.1097/MJT.0000000000001377; doi:10.1007/s43440-021-00245-z; @doi:10.1101/2021.04.30.21256415] effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low [@doi:10.1016/j.dsx.2021.102186] and these findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a].

Reviewer: ```suggestion
A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms [@doi:10.1001/jama.2021.3071], and although meta-analyses have reported both null [@doi:10.1101/2021.05.21.2125759; @doi:10.1101/2021.01.26.21250420] and beneficial [@doi:10.1016/j.dsx.2021.102186; @doi:10.1097/MJT.0000000000001402; @doi:10.1002/rmv.2265; @doi:10.31219/osf.io/k37ft; @doi:10.1097/MJT.0000000000001377; @doi:10.1007/s43440-021-00245-z; @doi:10.1101/2021.04.30.21256415] effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low [@doi:10.1016/j.dsx.2021.102186].
These findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a].
```

Thus, much like HCQ/CQ, enthusiasm for research that either has not or should not have passed peer review has led to large numbers of patients worldwide receiving treatments that might not have any effect or could even be harmful.  

> Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic [@url:https://www.researchsquare.com/article/rs-100956/v4], and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 [@url:https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals].

Reviewer: Was this the retracted version that was cited by DOI before?

Author: It is the same DOI, but the comments are on the landing page rather than in the document itself, so I wasn't sure whether citing the DOI made sense or whether it should be a URL!

Reviewer: Can we use the same citation for both of the retracted v4 versions?  The DOI version has better meta-data.  When I visit https://doi.org/10.21203/rs.3.rs-100956/v4, I see the comments at the bottom, so I think `doi:10.21203/rs.3.rs-100956/v4` will work here as well.

`$ manubot cite doi:10.21203/rs.3.rs-100956/v4 --format markdown`
1. **Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic**
Research Square Platform LLC *Research Square Platform LLC* (2020-10-30) <https://doi.org/gmhmfm>
DOI: [10.21203/rs.3.rs-100956/v4](https://doi.org/10.21203/rs.3.rs-100956/v4)
(manubot)

`$ manubot cite url:https://www.researchsquare.com/article/rs-100956/v4 --format markdown`
1. **Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic** (2021-07-14) <https://www.researchsquare.com>

```suggestion
Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic [@doi:10.21203/rs.3.rs-100956/v4], and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 [@url:https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals].
```

> Ivermectin is now being investigated by the large-scale TOGETHER clinical trial [@clinicaltrials:NCT04727424], which previously demonstrated no effect of HCQ and lopinavir-ritonavir [@doi:10.1001/jamanetworkopen.2021.6468], with preliminary results suggesting no effect of ivermectin [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].

Reviewer 2: ```suggestion
Ivermectin is now being investigated by the large-scale TOGETHER clinical trial [@clinicaltrials:NCT04727424], which previously demonstrated no effect of HCQ and lopinavir-ritonavir [@doi:10.1001/jamanetworkopen.2021.6468], with preliminary results suggesting no effect of ivermectin [@url:https://rethinkingclinicaltrials.org/news/august-6-2021-early-treatment-of-covid-19-with-repurposed-therapies-the-together-adaptive-platform-trial-edward-mills-phd-frcp].
Indeed, ivermectin is now the seventh treatment added to the Oxford University PRINCIPLE trial investigating potential COVID-19 therapeutics [@url:https://www.principletrial.org/news/ivermectin-to-be-investigated-as-a-possible-treatment-for-covid-19-in-oxford2019s-principle-trial]. 
```

Reviewer: @RLordan what's the narrative we're going for with this new sentence?  The original phrasing led me to believe we were going to conclude that ivermectin is ineffective.  Do we want to suggest there may still be a different outcome coming in the PRINCIPLE trial?  Or is the point something else?

Reviewer: This version reads well to me
